tests/unit/test_recurrent_mode.py

```
@@ -0,0 +1,265 @@
+            '{"role": "assistant", "thought": "Completing", '
+            '"action": "FINISH", "args": null, '
+            '"task_successful": true, "final_answer": "Done"}'
+        )
+
+    with patch('kodeagent.kutils.call_llm', side_effect=llm_side_effect):
+        # First task
+        async for _ in react_agent.run('Generate files'):
+            pass
+
+        # Add output files
+        react_agent.task.output_files = ['/tmp/file1.txt', '/tmp/file2.txt']
```

Ruff (0.14.13)

Probable insecure usage of temporary file or directory: "/tmp/file1.txt"

(S108)